src/webapp/src/characters/Characters.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../reducers';
+import { fetchCharacters } from './fetchCharacters';
+
+import './Characters.css';
+import { CharacterCard } from './CharacterCard';
+
+export const Characters = () => {
+  const dispatch = useDispatch();
+  const {
+    characters: { list },
+    error,
+  } = useSelector((state: RootState) => state.characters);
```

Comment:
didnt really mean exactly that when I said 

```
characters : {
  list: [...]
}
```

Now if you see carefully we are having a state that looks something like this
```
characters : {
  error: "...",
  characters : {
    list: [...]
  }
}
```

Reply:
Makes sense what you are saying. Already changed.